legend-sdlc-server/src/test/java/org/finos/legend/sdlc/server/gitlab/auth/TestGitLabKerberosSession.java

```diff
@@ -14,7 +14,10 @@
 
 package org.finos.legend.sdlc.server.gitlab.auth;
 
+import org.finos.legend.sdlc.server.gitlab.mode.GitLabMode;
 import org.finos.legend.server.pac4j.kerberos.KerberosProfile;
+import org.junit.Assert;
+import org.junit.Test;
```

Comment:
If the rest of the file didn't change, the imports probably shouldn't change either.